src/test/e2e/tests/specs.test.ts

```diff
@@ -136,6 +141,23 @@ describe('Finding a court or tribunal listing', () => {
     });
   });
 
+  describe('Idam SignIn selection', () => {
+    after(async () => {
+      await idamSigninPage.open('idam-signin');
+    });
+
+    it('should IdamSigIn header be exact text', async () => {
+      expect(await idamSigninPage.getPageTitle()).toEqual('Sign in to your account');
+    });
+
+    it('selecting crime and redirect to external url', async () => {
+      const valueToSelect = 'Crime';
+      await idamSigninPage.selectIdam(valueToSelect);
+      expect(await idamSigninPage.clickContinue()).toBeTruthy();
```

Comment:
This is a bit misleading, idamSigninPage.clickContinue function returns Promise with the type of SubscriptionManagementPage while actual action is taking user to the external page.